index.html

```diff
@@ -198,6 +198,49 @@ <h2 class="seventh-sheet__contact-me">Contact me</h2>
     </form>
       <div class="end-line"><img src="./images/end-line.png" alt="end-line"></div>
   </section>
+  <section class="desktop" id="desktop-modal">
```

Comment:
- [ ] The projects section should be created dynamically with the information from the projects array, please consider removing all the projects section code from your HTML and use JavaScript to create this section from ```line 48``` to ```line 143```